src/kOS/UserIO/TelnetWelcomeMenu.cs

```diff
@@ -152,7 +152,6 @@ private bool CPUListChanged()
                     if (newList[i] != availableCPUs[i])
                         itChanged = true;
 
-            Console.WriteLine("in CPUListChanged(): itChanged = {0}, newList.Count={1}, availableCPUs.Count={2}", itChanged, newList.Count, availableCPUs.Count);
```

Comment:
caught this when I did a scan through all the code for cases where there was a WriteLine, to see if any of them were leftover debug crud, and this one was.